src/week.py

```diff
@@ -0,0 +1,24 @@
+def week():
+    x = int(input("Enter week number (1-7) \n "))
+    if x == 1 :
+        print("Monday")
+    elif x == 2:
+        print("Tuesday")
+    elif x == 3:
+        print ("Wednesday")
+    elif x == 4:
+        print("Thursday")
+    elif x == 5:
+        print("Friday")
+    elif x == 6:
+        print("Saturday")
+    elif x == 7:
+        print("Sunday")
+
+week()
```

Comment:
Aggiungi un 
```python
if __name__ == "__main__":
    week()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do)